test/defn/test38.kore

```diff
@@ -0,0 +1,3078 @@
+// RUN: %interpreter
```

Comment:
This test was made to keep track of a bug we had in the modified decision tree generation.

Reply:
Ah so it doesn't actually need to run - is there a test added that will exercise the new features in that case? If not it would be good to have one

Reply:
There's an `imp-proof.kore` that runs the imp sum with proof output. It doesn't test any output yet, since the format is probably going to change a lot, but it's there to make sure it runs.

Reply:
Fine by me in that case! Thanks